src/main/java/com/flightstats/hub/webhook/WebhookLeader.java

```diff
@@ -262,12 +259,8 @@ private void send(ContentPath contentPath) throws InterruptedException {
                 long start = System.currentTimeMillis();
                 boolean shouldGoToNextItem = retryer.send(webhook, contentPath, webhookStrategy.createResponse(contentPath));
                 statsdReporter.time("webhook", start, "name:" + webhook.getName());
-                if (shouldGoToNextItem) {
-                    if (increaseLastUpdated(contentPath)) {
-                        if (!deleteOnExit.get()) {
-                            clusterCacheDao.setIfNewer(contentPath, webhook.getName(), WEBHOOK_LAST_COMPLETED);
-                        }
-                    }
+                if (shouldGoToNextItem && increaseLastUpdated(contentPath)) {
```

Comment:
This would read easier for me if "increaseLastUpdated" was name explicitly that it only conditionally updates last updated, which is implied because it returns a boolean, but still IMO, sorry, that name obviously predates your changes

Reply:
That's a small change and makes sense to do. 👍